src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java

```diff
@@ -557,7 +571,7 @@ public static DoneablePod createDoneablePodDefaultImpl(PodConfiguration config)
                     .addNewContainer()
                         .withName("selenium-node")
                         .withImage(config.getImage())
-                        .withImagePullPolicy("Always")
+                        .withImagePullPolicy(ImagePullPolicy.IfNotPresent.getPolicy())
```

Comment:
It should default to `Always`, but allow overriding to `IfNotPresent` otherwise you'll silently break other people relying on rolling tags.

Reply:
Gotcha, so make a field on `PodConfiguration.java` that includes an `imagePullPolicy` field that then defaults to `Always` in `KubernetesContainerClient.java`, unless overridden..

Not sure if I understand the whole flow after that; how would you map a given parameter that I(as a "user" of zalenium) provides it? Where would I practically override such variable from?

Reply:
I added another commit - hope I'm in the right direction(and that Windows+IntelliJ didn't mess up formatting too much).. I am unsure of how to actually read the environment variable(I guess, an enum similarly to the Resources one?) - Any help here would be appreciated. It passes the tests as-is, but obviously defaults to imagePullPolicy "Always" - always 😄. My Java is rusty, to be frank!

Reply:
- Committed a suggestion of how to do it. As said, my Java is a bit rusty, so feel free to rip it apart 👍